test/components/link-community/link-community.test.js

```diff
@@ -0,0 +1,53 @@
+import '../../../src/components/community/button-community.js';
+import '../../../src/components/community/link-community.js';
+import { expect, fixture, html } from '@brightspace-ui/testing';
+import { getDocumentLocaleSettings } from '@brightspace-ui/intl/lib/common.js';
+
+const langArticleMap = {
+	'en': 4545,
+	'ar-SA': 10489,
+	'es-MX': 9214,
+	'pt-BR': 9978,
+	'nl-NL': 7833,
+	'fr-CA': 6739
+};
+
+const langUrlMap = {
+	'en': 'https://community.d2l.com/brightspace/kb/articles/4545',
+	'ar-SA': 'https://community.d2l.com/brightspace-ar-SA/kb/articles/10489',
+	'es-MX': 'https://community.d2l.com/brightspace-es-MX/kb/articles/9214',
+	'pt-BR': 'https://community.d2l.com/brightspace-pt-BR/kb/articles/9978',
+	'nl-NL': 'https://community.d2l.com/brightspace-nl-NL/kb/articles/7833',
+	'fr-CA': 'https://community.d2l.com/brightspace-fr-CA/kb/articles/6739'
+};
+
+describe('community link', () => {
+	Object.keys(langArticleMap).map((langCode) => {
+		it(`Should have the right link for each lang code ${langCode}`, async() => {
+			getDocumentLocaleSettings().language = langCode;
+			const communityLink = await fixture(html`<d2l-labs-link-community type="link" text="Help" article-map=${JSON.stringify(langArticleMap)}></d2l-labs-link-community>`);
```

Comment:
Just pass the language into `fixture`:

```suggestion
			const communityLink = await fixture(html`<d2l-labs-link-community type="link" text="Help" article-map=${JSON.stringify(langArticleMap)}></d2l-labs-link-community>`, { lang: langCode });
```

Reply:
Neat!